UI/src/view/verificationWorkList.js

```diff
@@ -15,6 +15,14 @@
       });
     };
 
+    var addSpinner = function () {
```

Comment:
Why not palce this functions 'addSpinner' and 'removeSpinner' in workListView.js, since this file inherits from workListView.js and we just can call this functions